src/sentry/static/sentry/app/components/events/interfaces/utils.jsx

```diff
@@ -29,15 +29,17 @@ export function getCurlCommand(data) {
     result += ' \\\n -H "' + header[0] + ': ' + escapeQuotes(header[1] + '') + '"';
   }
 
-  switch (data.inferredContentType) {
-    case 'application/json':
-      result += ' \\\n --data "' + escapeQuotes(JSON.stringify(data.data)) + '"';
-      break;
-    case 'application/x-www-form-urlencoded':
-      result += ' \\\n --data "' + escapeQuotes(jQuery.param(data.data)) + '"';
-      break;
-    default:
-      result += ' \\\n --data "' + escapeQuotes(data.data) + '"';
+  if (data.data !== null && data.data !== undefined) {
```

Comment:
FYI: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/utils.jsx#L182-L184

Reply:
lodash is life.

Reply:
i think this might fix more than just JAVASCRIPT-2DQ, maybe look at similar issues there? nbd tho

Reply:
Hmm, don't see anything else in the similarity tab.